build/codegen/codegen.sh

```diff
@@ -38,5 +38,5 @@ bash ${CODE_GENERATOR}/generate-groups.sh \
     github.com/rook/rook/pkg/apis \
     "${GROUP_VERSIONS}" \
     --output-base "$(dirname "${BASH_SOURCE[0]}")/../../../../.." \
-    --go-header-file "${scriptdir}/boilerplate.go.txt" \
```

Comment:
In addition, let's make sure to `git rm boilerplate.go.txt`

Reply:
@BlaineEXE  wrote:
> In addition, let's make sure to `git rm boilerplate.go.txt`

I could not find a checked-in `boilerplate.go.txt `file.